### ISSUE TITLE
Remove persona support

### ISSUE THREAD

**Issue by hfiguiere** [photo/frontend#1583]
Persona is gonna be deprecated / shutdown within a year.

Removing mean that we should strongly suggest the user / admin stop using it.

**Comment by Dannyzen** [comment]
Hey @hfiguiere how would one go about disabling persona in the app and enabling alternative logins?

**Comment by hfiguiere** [comment]
more a question for @jmathai .

**Comment by jmathai** [comment]
@Dannyzen Persona is one of the options but it can be disabled via a configuration setting. Are you looking to remove it entirely wherever possible? I can remove it...

**Comment by Dannyzen** [comment]
@jmathai thanks for responding. I am looking for the definition of which configuration will disable it.

**Comment by jmathai** [comment]
@Dannyzen looks like I was incorrect. Logging in via a locally hosted database is optional ([see here](https://github.com/photo/frontend/blob/master/src/configs/defaults.ini#L12)) but persona is on by default and cannot be removed ([see here](https://github.com/photo/frontend/blob/master/src/html/assets/themes/fabrizio1.0/templates/login.php#L24)).

**Comment by cedricbonhomme** [comment]
Hello,

For your information I've made some minor modification on my instance (https://photos.cedricbonhomme.org).

And in fact, I almost never used the BrowserId authentication method. I'm trying to remove it:
https://github.com/cedricbonhomme/frontend/commits/cedric
If this can help.
No problem on my instance, but I need to test with a new setup.

**Comment by hfiguiere** [comment]
@cedricbonhomme do you mind submitting a PR for the persona/browserid removal? 

It should be noted that the proper way to do it would be to have it as a fallback to force setting the password (my instance only uses browserid) if it doesn't exists. When Mozilla finally sunset the server it can be permanently stripped.

I'll start with what you have done. Thanks !

**Comment by cedricbonhomme** [comment]
Yes, I can submit a PR. No problem. So you're thinking about having BrowserId as fallback? I think it is the case by default. I changed that: https://github.com/cedricbonhomme/frontend/commit/9e0811ba59a6ce0c7077011112834ae8c452b156#diff-cd7673220e591377b45ebab84ba79766L218

In fact on my branch, there are some personal adaptations for my instance. I have totally removed the support of browserID and Facebook login. I also have removed the link to the apple store.

**Comment by cedricbonhomme** [comment]
Initially I was also using BrowserId to log myself as admin. When a decided to switch to the local authentication method I had to edit the database manually. It was one year ago, I don't really remember... I think that I had to change the the column "owner" of the photos.

**Comment by Dannyzen** [comment]
@cedricbonhomme did you end up putting in a pr?

**Comment by cedricbonhomme** [comment]
No, not yet. As I explained on my instance I removed browserID and Facebook login methods.
But I can without problem send a pull request with only the removal of browserID.
I'll do that.

**Comment by cedricbonhomme** [comment]
I just deployed a new instance on my computer. Self-login is already the default when going to /setup for the first time.
It seems that I can simply remove the support of persona as I did on my instance.
